user-interface/src/components/CaseDetail.tsx

```diff
@@ -173,16 +173,72 @@ export const CaseDetail = (props: CaseDetailProps) => {
                       {caseDetail.debtor.address3}
                     </div>
                   )}
-                  {caseDetail.debtor.address4 && (
+                  {caseDetail.debtor.cityStateZipCountry && (
                     <div
-                      data-testid="case-detail-debtor-address4"
+                      data-testid="case-detail-debtor-cityStateZipCountry"
                       aria-label="debtor city, state, zip, country"
                     >
-                      {caseDetail.debtor.address4}
+                      {caseDetail.debtor.cityStateZipCountry}
                     </div>
                   )}
                 </div>
               </div>
+              <div className="debtor-counsel-information padding-bottom-4 case-card">
+                <h3>Debtor Counsel</h3>
+                {caseDetail.debtorAttorney && (
```

Comment:
We should either 1. Not show the header abover OR 2. Adda note that there is not debtor attorney. Or pro se.

Reply:
Phoenix: Show copy when the attorney is not available under the header. "Information is not available at this time."

Reply:
We should make this copy consistent for all info blocks n the detail screen where data is not present.